Edit Scripts/mteFunctions.pas

```diff
@@ -1264,25 +1266,16 @@ function SmallName(e: IInterface): string;
   ElementByIP:
   Element by Indexed Path
   
-  This is a function to help with getting at elements that are inside 
-  lists.  It allows you to use an "indexed path" to get at these elements
-  that would otherwise be inaccessible without multiple lines of code.
-  
-  Example usage:
-  element0 := ElementByIP(e, 'Conditions\[0]\CTDA - \Function');
-  element1 := ElementByIP(e, 'Conditions\[1]\CTDA - \Function');
+  This is a function Depreciated
 }
 function ElementByIP(e: IInterface; ip: string): IInterface;
-var
-  i, index: integer;
-  path: TStringList;
 begin
-  Result := ElementByPath(e, ip);
+    raise Exception.Create('Replace ElementByIP with ElementByPath');
```

Comment:
raising an exception?  doesn't this just mean users who update mteFunctions will get an error thrown?  why would we prefer this over just calling `ElementByPath` ?